corelib-schemaorg/src/main/java/eu/europeana/corelib/edm/utils/SchemaOrgUtils.java

```diff
@@ -432,6 +436,56 @@ public static void processOrganization(eu.europeana.corelib.definitions.edm.enti
         entityObject.setImage(organization.getFoafDepiction());
     }
 
+    /**
+     * Update properties of the given Schema.Org entity
+     * using data from the given EDM Timespan
+     *
+     * @param timespanObject Schema.Org Contextual Entity object to update
+     * @param timespan       source EDM timespan
+     */
+    public static void processTimespan(Timespan agent,
+                                    eu.europeana.corelib.edm.model.schemaorg.ContextualEntity timespanObject) {
+        // @id
+    	timespanObject.setId(agent.getAbout());
```

Comment:
for the time being the Timespans are mapped to things, simply call processConcept here

Reply:
please remove the commented out code and do not create dupplicate code between processConcept and processThing methods. The first method should simply call the later one.

Reply:
corrected as agreed.